src/os2datascanner/projects/admin/adminapp/templates/components/miniscanner/miniscan_results.html

```diff
@@ -32,7 +32,21 @@
             {% endfor %}
           {% endfor %}
         {% endfor %}
-      {% else %}
+      {% endif %}
+      {% if kle_results %}
+        {% for fileres in kle_results %}
+              <tr>
+                <td>{{ fileres.kle_res.file_path }}</td>
+                <td>
+                  {% for subres in fileres.kle_res.results %}
+                    <p>{{ subres }}</p>
+                  {% endfor %}
+                </td>
+                <td>{% trans "Classification" %}</td>
+              </tr>
+        {% endfor %}
+      {% endif %}
+      {% if not replies and not kle_results %}
```

Comment:
Perhaps this is me missing something, but why don't you just draw another table under the matches table if `kle_results` is set? :thinking: 

Here's my silly developer mockup of how that might look:

![Screenshot 2024-07-11 at 11-27-56 OSdatascanner](https://github.com/magenta-aps/osdatascanner/assets/45562185/de5dbbba-dd6c-4469-9ea5-03edc9f258bd)

Reply:
Yeah i suppose i could do that. as i said, i was waiting for feedback to know what to do with it. How can i access the points it has ? the classification function doesn't seem to return that if my memory is to be trusted.

Reply:
To quote from src/os2datascanner/engine2/commands/classify.py:

```python
            for classification, weight in engine.classify(fp.read())[:5]:
                print("\t", classification.ident, classification.label, weight)
```

Reply:
In that case, i will see what i can do to fix it. Thanks for mentioning and quoting